packages/core/src/utils.ts

```diff
@@ -104,11 +104,19 @@ import {
 import { getCreate3Address } from './config/utils'
 import { assertValidConfigOptions, parseConfigOptions } from './config/parse'
 import { SphinxRuntimeEnvironment, FailureAction } from './types'
-import {
-  SUPPORTED_NETWORKS,
-  SupportedChainId,
-  SupportedNetworkName,
-} from './networks'
+import { SUPPORTED_NETWORKS, SupportedChainId } from './networks'
+
+export const parseSemverVersion = (version: string) => {
```

Comment:
It appears this function would return a valid semver object for a string where the "v" is anywhere, e.g. "1.0v.2". This is b/c "replace" just finds the first instance of "v"

Reply:
I don't think that really needs to be handled by this function b/c the assumption is that this function is getting called with a `ValidManagerVersion` string since we perform validation on the input earlier in the deployment process. I do think the input type should be changed to `ValidManagerVersion` though.